Wikipedia/mw-utils/NSString+WMFPageUtilities.h

```diff
@@ -49,4 +49,11 @@ extern NSString* const WMFCitationFragmentSubstring;
  */
 - (NSString*)wmf_normalizedPageTitle;
 
+/**
+ *  Process a normalized title into one which can be used as an API request parameter.
+ *
+ *  @return The receiver, with spaces replaced by underscores and added percent escapes.
+ */
+- (NSString*)wmf_denormalizedPageTitle;
```

Comment:
so, the idea was to try to sync our code to the MW domain concept of a "normalized title," as seen when you send a request for a title that's not normalized.  for example, [sending a `mobileview` request for "Barack_Obama"](https://en.wikipedia.org/w/api.php?action=mobileview&page=Barack_Obama) has `"normalizedtitle": "Barack Obama"`.

That said, there are only two things our app _really_ cares about:

* Having a "canonical" title that we use when writing stuff to disk (i.e. don't end up w/ redundant "Barack Obama" and "Barack_Obama" folders)
* Adding percent escapes to an un-escaped title before sending to server

I think this means I can do away w/ "denormalizing" in this case.  However, we might want to chose one (normalized or not) when writing to disk, and enforce it—if that's not being done already.